### DISCUSSION TITLE
Fetch postgresql views

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7700] [Help] [answered]
Is there a way to fetch postgresql views into BB ?

I'm having this error message : 
<img width="328" alt="image" src="https://user-images.githubusercontent.com/28592069/189141842-78defd2a-b370-4dfd-a449-edb294274464.png">
But there is no way to add a PK constraint on a view in postgreSQL...

CC @mielune

**Comment by melohagan** [comment, marked as answer]
Hey @NicolasFabre81 

This is a known issue: https://github.com/Budibase/budibase/issues/6216
